Classes/TcaDataGenerator/FieldGenerator/TypePassword.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Empty line after declare

Reply:
There is no such rule in the coding style rules.